policy/release/sbom_cyclonedx.rego

```diff
@@ -192,8 +192,18 @@ _contains(needle, haystack) if {
 	needle_purl.namespace == hay_purl.namespace
 	needle_purl.name == hay_purl.name
 	_matches_version(needle_purl.version, hay)
+
+	not _excluded(needle_purl, object.get(hay, "exceptions", []))
 } else := false
 
+_excluded(purl, exceptions) if {
```

Comment:
I :heart: this pattern, we can add `_excluded` for any further pattern/key we have, nice